packages/opentelemetry-plugin-express/src/types.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { kPatched } from './express';
+import { Request } from 'express';
+
+export type PatchedRequest = { ['__ot_middlewares']?: string[] } & Request;
+
+// https://github.com/expressjs/express/blob/master/lib/router/index.js#L53
+export type ExpressRouter = {
```

Comment:
Don't know they existed since those are not the official types, i would prefer not to use them since they are not updated. Also since the official one doesn't contain internals i would prefer to keep those.